dev-environment.yml

```diff
@@ -52,4 +52,4 @@ dependencies:
     - noisyopt
 
     # To run CI against latest GeoUtils
-#     - git+https://github.com/GlacioHack/GeoUtils.git
+    - git+https://github.com/GlacioHack/geoutils.git
```

Comment:
Wasn't there a reason we left pulling from github? I think if we make a breaking change on geoutils, xdem will fail on new PRs without it being the "PR's fault". Any way around this? Otherwise, please make an issue for it so we can revert this asap!

Reply:
Yes, this should only be used to test that CI passes temporarily on the main branch for devs, we don't want to make a release with it! Should we add a reminder in `HOW_TO_RELEASE`?

Now that GeoUtils 0.0.15 is published, I can revert it back!